byway/src/main/webapp/script.js

```diff
@@ -12,19 +12,58 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- *
- */
-/* exported addRandomGreeting */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function initMap() {
+  var directionsService = new google.maps.DirectionsService();
+  var directionsRenderer = new google.maps.DirectionsRenderer();
+  var start = new google.maps.LatLng(37.7699298, -122.4469157);
+  var end = new google.maps.LatLng(37.7683909618184, -122.51089453697205);
+  var mapOptions = {
+    zoom: 14,
+    center: start
+  }
+  var map = new google.maps.Map(document.getElementById('map'), mapOptions);
```

Comment:
Generally speaking, it is almost always better to declare varaibles with `let` or `const `instead of `var`. The difference is that `var` is globally scope. This can lead to lots of confusing issues. You can read more about this here: https://www.javascripttutorial.net/es6/difference-between-var-and-let/

Now that everyone has gotten a decent handle on javascript, lets switch to using `let` in place of `var`.

Reply:
Ah okay, will use `let` from now on.